<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widgetGeoRefInfos</class>
 <widget class="QWidget" name="widgetGeoRefInfos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>587</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_worldFileData">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>125</y>
     <width>390</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>World file data</string>
   </property>
   <widget class="QLabel" name="label_4C_centerOfUpperLeftPixel_x">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>193</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_0A_scale_x">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_centerOfUpperLeftPixel_title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>164</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Center of upper left pixel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_scale_title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>61</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Scale</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4C_centerOfUpperLeftPixel_x_readOnly">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>190</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2B_rotationTerms_2_readOnly">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>220</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3E_scale_y_readOnly">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>117</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_rotationTerms_title">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>164</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>RotationTerms</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2B_rotationTerms_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>223</y>
      <width>26</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>#2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_0A_scale_x_readOnly">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>87</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_1D_rotationTerms_1_readOnly">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>190</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5F_centerOfUpperLeftPixel_y_readOnly">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>220</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5F_centerOfUpperLeftPixel_y">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>223</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_1D_rotationTerms">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>193</y>
      <width>26</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>#1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3E_yScale">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_status_aboutWorldFileData">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>370</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>---</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_EPSG_code">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>395</y>
     <width>390</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>EPSG Code</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_EPSGCode_value">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDialableCharactersOnly|Qt::ImhDigitsOnly|Qt::ImhLatinOnly</set>
    </property>
    <property name="text">
     <string>50000</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_EPSGCode_selectFromlist">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>select from list...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_setEPSGCode">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>set</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_status_aboutEPSGCode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>370</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>---</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_associatedName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>103</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;(name:...)&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineEdit_image1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>371</width>
     <height>25</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="inputMask">
    <string notr="true"/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_image2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>371</width>
     <height>25</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="inputMask">
    <string notr="true"/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_image3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>371</width>
     <height>25</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="inputMask">
    <string notr="true"/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_imageList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>5</y>
     <width>351</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Image(s):</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_0A_scale_x_readOnly</tabstop>
  <tabstop>lineEdit_3E_scale_y_readOnly</tabstop>
  <tabstop>lineEdit_4C_centerOfUpperLeftPixel_x_readOnly</tabstop>
  <tabstop>lineEdit_5F_centerOfUpperLeftPixel_y_readOnly</tabstop>
  <tabstop>lineEdit_1D_rotationTerms_1_readOnly</tabstop>
  <tabstop>lineEdit_2B_rotationTerms_2_readOnly</tabstop>
  <tabstop>lineEdit_EPSGCode_value</tabstop>
  <tabstop>pushButton_setEPSGCode</tabstop>
  <tabstop>pushButton_EPSGCode_selectFromlist</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
